2022-06-21 14:51:50,264 CRIT Supervisor is running as root.  Privileges were not dropped because no user is specified in the config file.  If you intend to run as root, you can set user=root in the config file to avoid this message.
2022-06-21 14:51:50,267 INFO supervisord started with pid 7
2022-06-21 14:51:51,269 INFO spawned: 'nginx' with pid 10
2022-06-21 14:51:51,271 INFO spawned: 'app' with pid 11
Bottle v0.12.21 server starting up (using WSGIRefServer())...
Listening on http://localhost:5000/
Hit Ctrl-C to quit.

2022-06-21 14:51:52,675 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2022-06-21 14:51:52,676 INFO success: app entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
[CTRL] POST /setup
[Time: 21.06.22 14:52:06] [Level: info] id: 1c021e849b6a8927
[Time: 21.06.22 14:52:06] [Level: info] coordinator: False
[Time: 21.06.22 14:52:06] [Level: info] clients: ['975457036f818710', '4d2aa209e6e9d824', '5c9b667c613a45ec', 'e533be9552d7c3ee', '1c021e849b6a8927', 'dd60a35071ef4645', '60d8d4918e546fbb', '2b311cb0d456fffb']
[Time: 21.06.22 14:52:06] [Level: info] state: initial
[Time: 21.06.22 14:52:06] [Level: info] transition: read
127.0.0.1 - - [21/Jun/2022 14:52:06] "POST /api/setup HTTP/1.0" 200 0
[CTRL] GET /status
127.0.0.1 - - [21/Jun/2022 14:52:06] "GET /api/status HTTP/1.0" 200 126
[Time: 21.06.22 14:52:07] [Level: info] state: read
[Time: 21.06.22 14:52:07] [Level: info] Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/FeatureCloud/app/engine/app.py", line 169, in guarded_run
    self.run()
  File "/usr/local/lib/python3.8/site-packages/FeatureCloud/app/engine/app.py", line 184, in run
    transition = self.current_state.run()
  File "/app/states.py", line 62, in run
    self.send_data_to_coordinator()
TypeError: send_data_to_coordinator() missing 1 required positional argument: 'data'

127.0.0.1 - - [21/Jun/2022 14:52:09] "GET /api/status HTTP/1.0" 200 133
[CTRL] GET /status
